SmartDeviceLinkTests/DevAPISpecs/SDLStreamingVideoLifecycleManagerSpec.m

```diff
@@ -568,7 +575,8 @@ __block void (^sendNotificationForHMILevel)(SDLHMILevel hmiLevel, SDLVideoStream
                         });
 
                         it(@"should not replace the existing screen resolution", ^{
```

Comment:
Please replace the test case block  "with missing screen height and screen width values" with the following:

```
context(@"with missing screen height and screen width values", ^{
    beforeEach(^{
        streamingLifecycleManager.preferredResolutions = @[];

        testVideoStartServicePayload = [[SDLControlFramePayloadVideoStartServiceAck alloc] initWithMTU:testMTU height:SDLControlFrameInt32NotFound width:SDLControlFrameInt32NotFound protocol:nil codec:nil];
        testVideoMessage = [[SDLV2ProtocolMessage alloc] initWithHeader:testVideoHeader andPayload:testVideoStartServicePayload.data];
        expect(@(CGSizeEqualToSize(streamingLifecycleManager.screenSize, CGSizeZero))).to(equal(@YES));
    });

    context(@"If no preferred resolutions were set in the data source", ^{
        beforeEach(^{
            streamingLifecycleManager.dataSource = nil;
            [streamingLifecycleManager handleProtocolStartServiceACKMessage:testVideoMessage];
        });

        it(@"should not replace the existing screen resolution", ^{
            expect(@(CGSizeEqualToSize(streamingLifecycleManager.screenSize, CGSizeZero))).to(equal(@YES));
            expect(streamingLifecycleManager.dataSource).to(beNil());
        });
    });

    context(@"If the preferred resolution was set in the data source", ^{
        __block SDLImageResolution *preferredResolutionLow = nil;
        __block SDLImageResolution *preferredResolutionHigh = nil;

        beforeEach(^{
            preferredResolutionLow = [[SDLImageResolution alloc] initWithWidth:10 height:10];
            preferredResolutionHigh = [[SDLImageResolution alloc] initWithWidth:100 height:100];

            streamingLifecycleManager.dataSource = testDataSource;
            streamingLifecycleManager.preferredResolutions = @[preferredResolutionLow, preferredResolutionHigh];

            [streamingLifecycleManager handleProtocolStartServiceACKMessage:testVideoMessage];
        });

        it(@"should set the screen size using the first provided preferred resolution", ^{
            const CGSize preferredFormat = CGSizeMake(preferredResolutionLow.resolutionWidth.floatValue, preferredResolutionLow.resolutionHeight.floatValue);
            expect(@(CGSizeEqualToSize(streamingLifecycleManager.screenSize, preferredFormat))).to(equal(@YES));
            expect(streamingLifecycleManager.dataSource).toNot(beNil());
        });
    });
});
```

Reply:
done